.travis.yml

```diff
@@ -0,0 +1,43 @@
+  - secure: Dlb4ITopzpn5vghVRlha4S49wOInlizyyjkt1wfwuZVPKSvMJsGX+8daKst7QqZfFW5lwc0rxaFUbNI2HcnOFKxzLjuFQ6Zn2rvqWlm0OcbtVs+1qPzkbz5X3o2TlLcDID0PnGAuWi+uYPDyb9OUIl/bzb0VtEubnBPy6ais0hPnR79sjjsaUbc67F+lt/aoGlNdxR8XAxo16ILt3TESlouEqj4NAH+rFw/s5JYsc9T+mYlAcCWXKJgBpShzjoEjOozGk+bCcwuEWnimbiW6g1mCCufgsgnUWNYTrA7ITBJknNCHYO3LrOE/x5hQri44m/gVd8y6MPFQzwkSa9Hqh8AO2hUmrxIhMsFHl2JysRvtcxLR8Nz9jGzQFZrVNGer/hEgxLYtx3AWq0zFpxI2WJLnteEzewT3xlBrBZhBZGWn6F9Q385xLQgmzpKXiAdT1XEUEPQ0vYErcSPFHF756EwoYU0NOgmWBnXmj48JiMSikwYUvXx/HWsns5sRY8mHGvc1dQBNOCOEINhc+f0jbT/x4TyvZ2uXodP3Dh833p4bXktQlUFgSZ4MV1AE7AN99y5nuM98abT8+g/fLqS/WCc36mpnvjjuR2X/vMvVJVEObL2YyNCHgNRYgEUJrS9+x7YKSGmeCvCN0kLY20p5SHyRCR0qpjWNCXsNXWcvin4=
+  - secure: TU4u7+YirlTCDAR5nBj0g6oRM1n+/j6kIS+9OwhEbwxZsnWJJi7cEWIQ3RdTczYC/eY2GwzYXOIUC8jDCX3NvkIEA00ev0lFZXqDX8qkqGD+ey1mqERTXs4LKpu2QP5ysxXWob3/+Jyw7l58l/Gxj6HFtikrgyGcORTjWr25cw9Es9y7F2+nyZDT5gA41OKOiTVb6WJYpVFKsHrflJRhqgXHqxtunygALo3nZn+p/RIx4wdofSC6H9KF8q8HQ4bFnqh3eUz2TtKNu8PxF3kJkG3vdTorIkN6nFjxj9HWNa1oMRqhrbYEDMiBPtcmHwm2KbEwo/DA5APx7SeHRBgUp6ZqCHq4EBCJP8JDsRt3EROXk4EQAVnRhVjLhZVJA8fgiJrDTPAGGpEeVFf1wsSveog+3myw1376JqYOC+u7gIiLI/fQP/0J5JMe4Dermk9J4aBEqnHTzvQPJq2m/OxijIDY7iIN9fOsvAawEBpGYPd0L9nO3iNymd6S6PJdmGqS+2yZ+c2EX2tZUOvkTz2x+t7jYsdB0pAGJhGF1/d7D5gdFc6W/1/ZghK6fKtDBKV+cPv5NCBxVpPD8X2E/qb4dUuaLpDqTWC8trw39hT5V9ezJU4YObTvz4Bnly2k0598O0IE63kuxT9ZI3W/JlZF2yOELArbh25sxQOnS3Taan4=
+  - secure: lFak7oRDvrUpzXkYEEHzFnPQiJhqnKx01oINSQirg/4KXnENzHKe1laz9dxi1l/nR6GSLyKcjkvUNUEaF6DKY4/Sv0OnivWLba9L6BebRw3+SC3NfakZS9DbTZKFMFS4F7BM4nuOEeSDZxn7Uab5Gx7Z4O73IMOmxtBkdK+YUth3iZJ9VVXaJkdaPurPmoKRxlg+ehIirwX6TX/ewpq1soOPH+CZ3HMxd6RRpxj0hp12Vwevh6/uDFJH4c34RG6SIR1kGN/r/gcq4vrkdz35YjbdhU0K52JQc27D8P5yhhP/THDh/EZS6Kxk1ou10P0yfm3hWNpp2Z+aO4wuokLTbVcygLxA+nMRy6/GioOOKM6e0iINjnz/c/RlCgImQeaWgHuXTjib0/M5tG3OpltDA5eDRhpGcarS26fzq4Sb8j3cP5C5MFuhj4k+rE0EQlzQqjnLEo8Wzo1f4KnYFW4SIRrNbWHHyorRERIw5STMX0JB8Xkit84MF20w3NMRQs6qDF6AkJVY/OUFUgupeXXjQaALHcQG74u2F+4bq38uiC387kKMzbA/b5N51ZQkeJipPjo5tMpS+HXU54QEbJAKkmavQhuBgCqLSUlRRxGt2l6f+Ny9AT53/WIMB3V3y9nZlr8glhyItcKEVVBAWd7d252U+6ylymx30REkDpYCHeQ=
+  - secure: D1mYdwt1Vz0CQbZJx/87AgmKIGV1qdaDPGABoSz5YchS8NrsFXkcty+WfytpiBfWqx/8mXaT0Y6MUcl888L6gRoGSVjRsWu7l5KhchiRTStW63frm1VkClvJIk5PTZvyH8S7XJNJ7TjqUhtlXk3BsMzFSqor456ssc7rpEI+WIdRmV4VwMuIXe35W/mJs3kXEJXTks2DkUSzjKYD8yOQA7mnsv2J/q0WaPDZGagAD7AMH89/Tveku970UvrbOIIBpzhA4TA04OW/d1pwotod/lZnTDp8N722yGaKeCO+ehw/IolMNLY/ivKWyZxfibP9TUaTOY1h0lwLj2VmixAvlSZdwmfFDO69mbB8tRQ3aNI6bX7gmWTnGcPh2aZ6cnHPrZ43C2jVAP0YOHZ7Rmc+U22wHyfgHsPmP36t2mJrN3AjB8vhcY10g9GuhRQ9AHX5A8Os5jET/7RoC0o03xBCXc+fTngkASyaSmavnIhZK3pUs66fH71GOE0IujtvhtvMJFOWn5mxEWUURfyZ+X/bRoG5+Q+sFgSH41humJEc+PDfj3AiyVtmYYve1QD1xeKW1UDAtCkhV0gZCeGrhloDRaWUDFh3svRSJdW6R8+QJf327NnLk2kBVLuKQTOUGgTeo+rLbk6EKc3kWBCxG8iYX6LBqDUIiQyxA9YwpjWYnAo=
+
+stages:
+  - test
+  - name: deploy
+    if : branch = master
```

Comment:
Worth noting that e.g. this Pull Request is viewed as against the master branch, so this stage still runs and we wouldn't want to deploy a PR until it's been reviewed & merged ([Docs on conditional build stages](https://docs.travis-ci.com/user/conditional-builds-stages-jobs/) say that for a PR this branch is considered to be the base branch (i.e. master here))
You can see that [the relevant Travis build for this PR](https://travis-ci.com/github/JoeWFisher/DevOps-Course-Starter/jobs/480990618) has indeed done the deployment step.